### ISSUE TITLE
timeout option + 0 animation speed = exception on manual close

### ISSUE THREAD

**Issue by rosenfeld** [needim/noty#239]
Try this:

``` javascript
$.noty.defaults.animation.speed = 0
noty({text: 'test', timeout: 10000})
$.noty.closeAll()
```

I'm using try-catch for now when using `close` and `closeAll` until this bug is fixed but I think you'd like to know about it.

**Comment by siddharthbhagwan** [comment]
`$.noty.closeAll()` isn't throwing an error. However, the noty does continues to stick forever.

**Comment by rosenfeld** [comment]
Maybe the behavior changed with the latest version.

**Comment by needim** [comment]
fixed in v2.3.11
